awscli/customizations/s3/filegenerator.py

```diff
@@ -173,6 +173,12 @@ def list_files(self, path, dir_op):
         if not self.should_ignore_file(path):
             if not dir_op:
                 size, last_update = get_file_stat(path)
+                if last_update is EPOCH_TIME:
+                    warning = PrintTask(
```

Comment:
`create_warning` will say the file is skipped, which is not the case here. Didn't seem like there was much value in creating a new function that doesn't really abstract anything.